Homework_2/UserActions/FibonacciNumbersUserAction.cs

```diff
@@ -0,0 +1,75 @@
+namespace Homework_2.UserActions;
+
+public class FibonacciNumbersUserAction : IUserAction
+{
+    public void MakeAction()
+    {
+        while (true)
+        {
+            Console.ForegroundColor = ConsoleColor.Blue;
+            Console.Write("Введите число Фибоначчи: ");
+            Console.ForegroundColor = ConsoleColor.White;
+        
+            if (int.TryParse(Console.ReadLine(), out int fibbonachiNumber) && fibbonachiNumber >= 0)
```

Comment:
be consistent. too many different Fibonacci names)

Reply:
++